examples/Shared/Enums/ConnectionEnvironment.swift

```diff
@@ -21,13 +21,32 @@ enum ConnectionEnvironment: String, EnvironmentSettings {
 extension MIAWConfigurationStore {
     var environments: [String: ConnectionConfigurationModel] {
         get {
-            guard let dictionary = [String: ConnectionConfigurationModel](rawValue: userDefaults.string(forKey: Keys.environments.rawValue) ?? "") else {
```

Comment:
Moved this to json decoding so that when we update the base model it wont fail to map to the default, if we use the json decoding it will allow and unknown or unused value to set a default and wont break the settings page.